code/game/machinery/cryopod.dm

```diff
@@ -303,6 +303,12 @@
 				preserve = 1
 				break
 
+		if(istype(W,/obj/item/device/pda))
+			var/obj/item/device/pda/P = W
+			if(P.id)
+				qdel(P.id)
```

Comment:
may want to make `P.id = null` just to be safe and ensure the PDA and ID both GC properly.